BlackJack.py

```diff
@@ -0,0 +1,147 @@
+from random import choice
+from enum import Enum, auto
+from UserInputProvider import *
+
+
+class AutoName(Enum):
+    def _generate_next_value_(name, start, count, last_values):
+        return name
+
+
+class Color(AutoName):
+    HEART = auto()
+    DIAMOND = auto()
+    PEAK = auto()
+    CLUB = auto()
+
+
+class Sign(AutoName):
+    TWO = auto()
+    THREE = auto()
+    FOUR = auto()
+    FIVE = auto()
+    SIX = auto()
+    SEVEN = auto()
+    EIGHT = auto()
+    NINE = auto()
+    TEN = auto()
+    JACK = auto()
+    QUEEN = auto()
+    KING = auto()
+    ACE = auto()
+
+
+class Value(Enum):
+    ONE = 1
+    TWO = 2
+    THREE = 3
+    FOUR = 4
+    FIVE = 5
+    SIX = 6
+    SEVEN = 7
+    EIGHT = 8
+    NINE = 9
+    TEN = 10
+    ELEVEN = 11
+
+
+class Card(object):
+
+    def __init__(self, sign, color, value):
+        self.card = [str(sign) + " " + str(color), value]
+
+
+class Desk(object):
+
+    def __init__(self):
+        self.cards = self.list_of_cards()
+        self.waist = {}
+        for i in range(len(self.cards)):
+            self.waist[self.cards[i][0]] = self.cards[i][1]
+
+    @staticmethod
+    def list_of_cards():
+        cards = []
+        for color in Color:
+            cards_in_colour = [Card(Sign.TWO.value, color.value, Value.TWO.value).card,
+                               Card(Sign.THREE.value, color.value, Value.THREE.value).card,
+                               Card(Sign.FOUR.value, color.value, Value.FOUR.value).card,
+                               Card(Sign.FIVE.value, color.value, Value.FIVE.value).card,
+                               Card(Sign.SIX.value, color.value, Value.SIX.value).card,
+                               Card(Sign.SEVEN.value, color.value, Value.SEVEN.value).card,
+                               Card(Sign.EIGHT.value, color.value, Value.EIGHT.value).card,
+                               Card(Sign.NINE.value, color.value, Value.NINE.value).card,
+                               Card(Sign.TEN.value, color.value, Value.TEN.value).card,
+                               Card(Sign.JACK.value, color.value, Value.TEN.value).card,
+                               Card(Sign.QUEEN.value, color.value, Value.TEN.value).card,
+                               Card(Sign.KING.value, color.value, Value.TEN.value).card,
+                               Card(Sign.ACE.value, color.value, [Value.ONE.value, Value.ELEVEN.value]).card]
+            cards.extend(cards_in_colour)
+        return cards
+
+
+class Board(object):
+
+    def __init__(self):
+        self.desk = Desk().waist
+        self.player_hand = []
```

Comment:
to sa wlasnosci gracza i krupiera. stol co najwyzej moze printowac ich rece. Ale to gracz ma swoje karty